### ISSUE TITLE
During first run all ID's are equal, which is against HTML5 WCAG 2.1 A 4.1.1

### ISSUE THREAD

**Issue by janhoogeveen** [roginfarrer/react-collapsed#52]
Looking at the source code, the initial random value of collapsed components is 0. When rendering multiple collapse component they all share the same ID, which is against HTML5 WCAG 2.1 A 4.1.1.

```<div id="react-collapsed-panel-0"></div>```

Should the unique ID be set at initialization of the hook, instead of with useEffect after mounting?

**Comment by roginfarrer** [comment]
Hi @janhoogeveen, could you provide a codesandbox that reproduces this issue? Every instance of the component should have a unique id

**Comment by 7iomka** [comment]
https://github.com/roginfarrer/react-collapsed/blob/master/src/utils.ts#L132

I see the same issue, this is problem in SSR projects

**Comment by roginfarrer** [comment]
@7iomka @janhoogeveen Can you check if this PR fixes your issues? https://github.com/roginfarrer/react-collapsed/pull/54

**Comment by janhoogeveen** [comment]
@roginfarrer How would I do that? If I try to install this with a github link in my package.json I'm getting duplicate react warnings.

**Comment by roginfarrer** [comment]
You can install specific branches, it's a pretty neat feature:

```sh
yarn add roginfarrer/react-collapsed#fix-unique-ids-on-first-render
npm install roginfarrer/react-collapsed#fix-unique-ids-on-first-render
```

**Comment by janhoogeveen** [comment]
> You can install specific branches, it's a pretty neat feature:
> 
> ```shell
> yarn add roginfarrer/react-collapsed#fix-unique-ids-on-first-render
> npm install roginfarrer/react-collapsed#fix-unique-ids-on-first-render
> ```

I'll take a look, NPM is down right now but later today or tomorrow.

**Comment by janhoogeveen** [comment]
When I remove my lockfiles and replace `"react-collapsed": "^3.0.0"` with `"react-collapsed": "roginfarrer/react-collapsed#fix-unique-ids-on-first-render"` in package.json I get the following error when trying to run my application:

```
Module not found: Can't resolve 'react-collapsed' in '/Users/jan/Projects/REDACTED/src/components/collapse'
Error: Hook can only be invoked from render methods.

    at Object.n.options.__h (/Users/jan/Projects/REDACTED/node_modules/preact/debug/dist/debug.js:1:6026)
    at v (/Users/jan/Projects/REDACTED/node_modules/preact/hooks/dist/hooks.js:1:152)
    at Object.s [as useReducer] (/Users/jan/Projects/REDACTED/node_modules/preact/hooks/dist/hooks.js:1:307)

```

I think this happens because the `dist` folder does not exist in my `node_modules` version of `react-collapsed` when I use the branch name to install it.

**Comment by roginfarrer** [comment]
I went ahead and merged and released the fix in 3.0.1, since it should be harmless. Let me know how it goes
